docs/book/user-guide/best-practices/quick-wins.md

```diff
@@ -0,0 +1,449 @@
+log_metadata({"lr": 1e-3, "epochs": 10, "prompt": my_prompt})
+
+# Group related metadata in categories for better dashboard organization
+log_metadata({
+    "training_params": {
+        "learning_rate": 1e-3,
+        "epochs": 10,
+        "batch_size": 32
+    },
+    "dataset_info": {
+        "num_samples": 10000,
+        "features": ["age", "income", "score"]
+    }
+})
+
+# Use special types for consistent representation
+from zenml.metadata.metadata_types import StorageSize, Uri
+log_metadata({
+    "dataset_source": Uri("gs://my-bucket/datasets/source.csv"),
+    "model_size": StorageSize(256000000)  # in bytes
+})
+```
+
+**Works at multiple levels:**
+- **Within steps**: Logs automatically attach to the current step
+- **Pipeline runs**: Track environment variables or overall run characteristics
+- **Artifacts**: Document data characteristics or processing details
+- **Models**: Capture hyperparameters, evaluation metrics, or deployment information
+
+**Best practices:**
+- Use consistent keys across runs for better comparison
+- Group related metadata using nested dictionaries
+- Use ZenML's special metadata types for standardized representation
+
+*Metadata becomes the foundation for the Experiment Comparison tool and other
+dashboard views.* (Learn more: [Metadata |
+ZenML](https://docs.zenml.io/concepts/metadata), [Tracking Metrics with
+Metadata](https://docs.zenml.io/how-to/model-management-metrics/track-metrics-metadata))
+
+## 2 Activate the **Experiment Comparison** view (ZenML Pro)
+
+**Why** -- side-by-side tables + parallel-coordinate plots of any numerical metadata help you quickly identify patterns, trends, and outliers across multiple runs. This visual analysis speeds up debugging and parameter tuning.
+
+**Setup** -- once you've logged metadata (see quick win #1) nothing else to do; open **Dashboard → Compare**.  
+
+[![Experiment Comparison Video](../.gitbook/assets/experiment_comparison_video.png)](https://www.loom.com/share/693b2d829600492da7cd429766aeba6a)
+
+**Compare experiments at a glance:**
+- **Table View**: See all runs side-by-side with automatic change highlighting
+- **Parallel Coordinates Plot**: Visualize relationships between hyperparameters and metrics
+- **Filter & Sort**: Focus on specific runs or metrics that matter most
+- **CSV Export**: Download experiment data for further analysis (Pro tier)
+
+**Practical uses:**
+- Compare metrics across model architectures or hyperparameter settings
+- Identify which parameters have the greatest impact on performance
+- Track how metrics evolve across iterations of your pipeline
+
+(Learn more: [Metadata | ZenML - Bridging the gap between ML & Ops](https://docs.zenml.io/how-to/model-management-metrics/track-metrics-metadata), [New Dashboard Feature: Compare Your Experiments - ZenML Blog](https://www.zenml.io/blog/new-dashboard-feature-compare-your-experiments))
+
+## 3 Drop-in **Experiment Tracker Autologging**
```

Comment:
im not sure this works for all the trackers but if it does id put in one concrete code example at least